.github/workflows/conda-lock.yml

```diff
@@ -71,15 +88,21 @@ jobs:
             --file "${python_version_file}" \
             --platform win-64 \
             --filename-template "${template}" \
+            -e bigquery \
```

Comment:
if you want to add `duckdb` here, it's available on conda-forge now, so this should work.

Reply:
Huzzah!

Reply:
Done!